src/aat/java/uk/gov/hmcts/ccd/datastore/befta/DataStoreTestAutomationAdapter.java

```diff
@@ -182,6 +182,8 @@ public Object calculateCustomValue(BackEndFunctionalTestScenarioContext scenario
             }
         } else if (key.toString().equalsIgnoreCase("dateTwentyDaysFromToday")) {
             return LocalDate.now().plusDays(20).toString();
+        } else if (key.toString().equalsIgnoreCase("generateUUID")) {
```

Comment:
Not part of this change, but this method needs some serious refactoring !

Reply:
agreed.